app/controllers/api/v1/github_users_controller.rb

```diff
@@ -59,7 +59,7 @@ def compute_score_for_user(other_users_ids)
 
   def other_team_members_ids
     if permitted_params[:froggo_team] == "true"
-      team = froggo_team
+      team = FroggoTeam.find(permitted_params[:team_id])
```

Comment:
hmm! a qué se debe el cambio? no se si cachas la diferencia entre `find` y `find!`. `find!` te levanta la exepción cuando ocurre el error, mientras que `find` sigue adelante nomás. Si es que siempre viene el `team_id` estaría bien, sino, creo que deberia haber algun `if` por ahi para que no se caiga todo.

Reply:
es que no me funciona hacer .find! no se por que 
todo el rato me dice .... did you mean find?

Reply:
estaba puro leseando, no existe `find!`, vimos por interno que el que existe es `find_by!`